content/influxdb/cloud-iox/having.md

```diff
@@ -0,0 +1,78 @@
+---
+title: The HAVING clause
+list_title: The HAVING clause
+description: > 
+    Use the `HAVING` clause to filter query results based on a spcified condition.
+menu:
+  influxdb_cloud_iox:
+    name: The HAVING clause
+    parent: Explore data using SQL
+weight: 250
+---
+
+The `HAVING` clause places conditions on results created by an aggregate or group.  The `HAVING` clause must follow the `GROUP BY` clause and precede the `ORDER BY` clause. Whereas the `WHERE` clause filters rows based on a specified condition, the `HAVING` clause filters based on **groups of rows** with a specified  condition.
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+### Syntax
+
+```sql
+
+SELECT_clause FROM_clause [WHERE_clause] [GROUP_BY_clause] [HAVING_clause] [ORDER_BY_clause] 
+
+--Example
+SELECT a, b, MEAN(c) 
+FROM measurement
+GROUP BY a, b 
+HAVING MEAN(c) > 10
+```
+
+### Examples
+
+Group results based on a specified function:
+
+```sql
+SELECT MEAN("water_level") AS "mean_water_level", "location", "time"
+FROM "h2o_feet" 
+GROUP BY "location","time"
+HAVING MEAN("water_level") > 9.8
+ORDER BY "time"
+```
+Results:
+
+| location     | mean_water_level | time                     |
+| :----------- | :--------------- | :----------------------- |
+| coyote_creek | 9.816            | 2019-08-28T06:54:00.000Z |
+| coyote_creek | 9.862            | 2019-08-28T07:00:00.000Z |
+| coyote_creek | 9.902            | 2019-08-28T07:06:00.000Z |
+| coyote_creek | 9.938            | 2019-08-28T07:12:00.000Z |
+| coyote_creek | 9.957            | 2019-08-28T07:18:00.000Z |
+| coyote_creek | 9.964            | 2019-08-28T07:24:00.000Z |
+
+The query groups results from a `MEAN()` function having a water_level greater than 9.8 feet. This is a partial data set.
```

Comment:
~~~suggestion
### Return rows with an aggregate value greater than a specified number

```sql
SELECT
  MEAN("water_level") AS "mean_water_level", "location"
FROM
  "h2o_feet" 
GROUP BY
  "location"
HAVING
  "mean_water_level" > 5
```

{{< expand-wrapper >}}
{{% expand "View example results %}}
The query returns on rows with values in the `mean_water_level` greater than 5 _after_ the aggregate operation.

| location     | mean_water_level  |
| :----------- | :---------------- |
| coyote_creek | 5.359142420303919 |
{{% /expand %}}
{{< /expand-wrapper >}}
~~~